File: src/DutchMetar.Web/DutchMetar.Web.Shared/Models/AirportSummary.cs
namespace DutchMetar.Web.Shared.Models;

public class AirportSummary
{
    public required string Icao  { get; set; }
    
    public DateTime LastIssuedMetarDate { get; set; }
    
    public bool IsAuto { get; set; }
    
    public bool IsCavok { get; set; }
    
    public int? WindDirection { get; set; }
    
    public int? WindSpeedKnots { get; set; }
}

File: src/DutchMetar.Web.Client/src/app/shared/models/airport/airport-summary.ts
import { MeteoCondition } from '../../types/meteo-condition';

export type AirportSummary = {
    icao: string;
    lastIssuedMetarDate: string;
    isAuto: boolean;
    isCavok: boolean;
    isCorrected: boolean;
    meteoCondition: MeteoCondition
    windDirection?: number;
    windSpeedKnots?: number;
};
